README.md

```
@@ -50,18 +50,18 @@ Supports looping on items. You can choose whether to enable looping on `days`, `
-
 ### Flat Scroll Wheel
 
 |    Flat Holo Overlay    |    Flat Highlight Overlay    |    Flat Line Overlay    |
 | :---------------------: | :--------------------------: | :---------------------: |
 | ![](demo/flat_holo.gif) | ![](demo/flat_highlight.gif) | ![](demo/flat_line.gif) |
 
+### Curve Scroll Wheel
+
+|    Curve Holo Overlay    |    Curve Highlight Overlay    |    Curve Line Overlay    |
+| :----------------------: | :---------------------------: | :----------------------: |
+| ![](demo/curve_holo.gif) | ![](demo/curve_highlight.gif) | ![](demo/curve_line.gif) |
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)